virtual/README.md

```diff
@@ -8,18 +8,17 @@ Set up a virtual cluster with DeepOps. Useful for...
 
 ## Prerequisites
 
-Running DeepOps virtually assumes that the host machine's OS is Ubuntu 16.04 or greater. If this is
-not the case, the `bootstrap_virtual.sh` and `cluster_up.sh` scripts may be modified to work with
-a different OS.
+Approved OS:
 
-Also, using VMs and optionally GPU passthrough assumes that the host machine has been configured to
-enable virtualization in the BIOS. For instructions on how to accomplish this, refer to the sections
-at the bottom of this README: `Enabling virtualization and GPU passthrough`.
+Ubuntu 16.04 (or greater)
+CentOS 7.6 (or greater)
```

Comment:
These should be bulleted (use a `* ` before each one). Also need an additional newline after the second bullet so that there is space for the next paragraph.